docs/content/author-bundles.md

```diff
@@ -31,17 +31,21 @@ although Porter does have extra fields that are specific to making Porter bundle
 name: azure-wordpress
 description: Install Wordpress on Azure
 version: 0.1.0
-tag: getporter/azure-wordpress
+registry: getporter
+reference: getporter/azure-wordpress:v0.1.0
 dockerfile: dockerfile.tmpl
 ```
 
 * `name`: The name of the bundle
 * `description`: A description of the bundle
 * `version`: The version of the bundle, uses [semver](https://semver.org). Should not have a 'v' prefix.
-* `tag`: The tag to use when the bundle is published to a registry. The format is `REGISTRY/IMAGE`, or optionally `REGISTRY/IMAGE:TAG` 
-   if you do not want the default behavior of defaulting the docker image TAG portion to the version of the bundle. The invocation
-   image name will be based on this value.  For example, if the bundle `tag` is `getporter/porter-hello` and the `version` is `0.1.0`,
-   then the invocation image name will be `getporter/porter-hello-installer:v0.1.0`
+* `registry`: The registry to use for publishing the bundle. The format is `REGISTRY/ORG`.
+    Both the final bundle reference and invocation image name will be based on this value.
+    For example, if the bundle name is `porter-hello`, registry is `getporter` and the version is `0.1.0`,
+    the bundle reference will be `getporter/porter-hello:v0.1.0` and the invocation image name will be `getporter/porter-hello-installer:v0.1.0`
+* `reference`: OPTIONAL. The full bundle reference which takes precedence over any values set for the `registry`, `name` and `version` fields.
```

Comment:
Defaulting the Docker tag portion of the `reference`, when not provided (e.g. `reference: getporter/wordpress`) will work just like it did for `tag`, i.e. the `version` value will be used.  If `version: 0.1.0`, the final bundle ref will be `getporter/wordpress:v0.1.0`.

Should I add a clarification here around this scenario?

Reply:
Yeah I think mentioning that it will automatically append the version as the TAG, maybe say that we recommend not specifying the TAG part of `REGISTRY_HOST/ORG/BUNDLE:TAG` and let porter fill it in?